packages/fxa-settings/src/pages/Signup/SignupConfirmed/index.tsx

```diff
@@ -20,7 +21,9 @@ const SignupConfirmed = ({
   isSignedIn,
   serviceName,
 }: SignupConfirmedProps & RouteComponentProps) => (
-  <Ready {...{ continueHandler, isSignedIn, viewName, serviceName }} />
+  <AppLayout>
```

Comment:
Do we want to pass in a page `title` in `AppLayout` or wait for [FXA-6803](https://mozilla-hub.atlassian.net/browse/FXA-6803?atlOrigin=eyJpIjoiNmE3YzJjM2VjNGU4NGQxOGJlZDllZDdlNTc5MmQ1MzEiLCJwIjoiaiJ9)?

Reply:
I'll wait and add a note to FXA-6803!